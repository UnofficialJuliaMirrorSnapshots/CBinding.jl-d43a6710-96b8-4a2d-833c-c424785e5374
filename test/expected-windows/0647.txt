4: 00000000
x: 05000000
l: 05000000
