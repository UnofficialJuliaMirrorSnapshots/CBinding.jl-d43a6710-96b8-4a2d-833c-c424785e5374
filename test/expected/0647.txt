8: 0000000000000000
x: 0500000000000000
l: 0500000000000000
